pkgs/tools/bootloaders/limine/default.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs, lib, fetchurl, llvm } :
+
+pkgs.stdenv.mkDerivation rec {
+  name = "limine";
+  version = "4.20221006.1";
+  src = fetchurl {
```

Comment:
Please use `fetchFromGitHub` here

Reply:
I used `fetchurl` here because I need the release tarball instead of the repository itself. The tarballs have all the dependencies needed for building the package.